src/main/java/org/interledger/ilp/exceptions/InterledgerQuotingProtocolException.java

```diff
@@ -2,6 +2,19 @@
 
 public class InterledgerQuotingProtocolException extends Exception {
```

Comment:
@earizon & @adrianhopebailie: I submitted [IL-RFC-#168](https://github.com/interledger/rfcs/issues/188) to remove the `data` field in the `InterledgerError`, or else clarify its usage.

IMHO, it doesn't make sense to include a field that _could_ mean anything (ILQP info, or just debug info, or nothing).  Instead, if sub-protocols are going to specify data that is added into an `InterledgerError`, then they should be specifying it.

At any rate, would appreciate any feedback, here or in that ticket.